### PR TITLE
Update to Manage Subuser, added v3 API link

### PR DESCRIPTION
**Pull request by hmbrown64**
Updated information:

Subusers can be managed via [Subuser API](https://sendgrid.api-docs.io/v3.0/subuser-api) or by logging into your SendGrid account under Settings > Subuser Management.

To manage a Subuser, click the name of the Subuser. This will take you to the Subuser’s account and profile settings. From there, you can manage and update your Subuser Settings.

https://github.com/sendgrid/docs/issues/2747

**Description of the change**:
**Reason for the change**:
**Link to original source**:

@ksigler7

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/docs?pullRequest=2763) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/docs?pullRequest=2763) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/docs?pullRequest=2763) it.</sub>

File: source/User_Guide/Settings/Subusers/index.html [hunk 1]
```diff
@@ -20,7 +20,9 @@
 Manage Subusers
 {% endanchor %}
 
-To manage a subuser, click the name of the subuser you would like to manage. This will take you to the subuser’s profile page. From there, you can manage and see quite a bit of information about the subuser you selected.
+Subusers can be managed via [Subuser API](https://sendgrid.api-docs.io/v3.0/subuser-api) or by logging into your SendGrid account under Settings > Subuser Management.
```
Comment (ksigler7) [inline source/User_Guide/Settings/Subusers/index.html]: Oooh, I figured out the issue! This is one of the odd .html pages, so the link does need to be in the `<a href...>` form. When you get a chance, do you mind fixing this link, and the other broken one too (on line 31)

**Comment by ksigler7** [comment]
Can you also please add your full name, a picture, and that you work for SendGrid to your github account? :)

**Comment by thinkingserious** [comment]
Hello @hmbrown64,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**